src/Microsoft.DotNet.GitHub.IssueLabeler/Models/Labeler.cs

```diff
@@ -236,15 +238,22 @@ private async Task InnerTask(string owner, string repo, int number)
                 }
 
                 // if newlabels has no area-label and existing does not also. then comment
-                if (!foundArea && issueMissingAreaLabel && labelRetriever.CommentWhenMissingAreaLabel)
+                if (!foundArea && issueMissingAreaLabel)
                 {
-                    if (issueOrPr == GithubObjectType.Issue)
+                    if (!string.IsNullOrEmpty(options.NoAreaDeterminedLabel))
                     {
-                        await _gitHubClientWrapper.CommentOn(owner, repo, iop.Number, labelRetriever.MessageToAddAreaLabelForIssue);
+                        await _gitHubClientWrapper.AddLabels(owner, repo, iop.Number, new[] { options.NoAreaDeterminedLabel });
                     }
-                    else
+                    else if (labelRetriever.CommentWhenMissingAreaLabel)
```

Comment:
Just to clarify, in cases where the new label is enabled, we don't want to continue adding the comment too? It seems like that was what was suggested here: https://github.com/dotnet/issue-labeler/issues/27#issuecomment-1430532166, but I wanted to make sure this was an intended change.

I think the logic in that comment is reasonable, as long as people are able to rewire their own thought processes to treat the label with the same "urgency" as the comment. Notably, the label won't come with a message that directly requests the addition of an area label.

Reply:
Actually, it looks like there was consensus that the comment causes more confusion than it helps. We can always add an "opt in" option later if some repos want the comment _and_ the new label.

Reply:
Thanks; that was my thought process as well. _And_ laying down the comment in reaction to the label getting applied could be done through FabricBot instead of relying on issue-labeler to handle that directly. That would bring symmetry to how `area-` labels getting applied trigger issue tagging/comments.